src/main/resources/org/jenkinsci/plugins/ewm/nodes/NodeDiskPool/config.jelly

```diff
@@ -6,4 +6,9 @@
     <f:entry>
         <f:repeatableProperty field="nodeDisks" add="${%Add Disk}" header="${%Disk}"/>
     </f:entry>
+    <f:entry>
+        <div align="right">
+            <f:repeatableDeleteButton value="${%Delete Disk Pool}"/>
```

Comment:
Ideally it should be in the calling jelly. In such case you would be able to use NodeDiskPool definitions in other cases if required. Example: https://github.com/jenkinsci/github-branch-source-plugin/blob/900ea989a624223c88bddb3f1fbb3a3bd2193b5f/src/main/resources/org/jenkinsci/plugins/github_branch_source/GitHubConfiguration/config.jelly#L5-L11